flutter/rsvp/lib/Views/eventDetailsRespondent.dart

```diff
@@ -0,0 +1,63 @@
+import 'package:flutter/material.dart';
```

Comment:
We already have `eventDetails` page which Grey is working on.
The difference between `eventDetails` page for created events and responded event is visibility of buttons (not more)
So, we can set visibility of buttons based on properties of `eventDetailsViewModel` (which is connected to the repository and aware of two lists)